frontend/src/metabase/nav/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -39,6 +41,27 @@ export const AdminNavbar = ({
   adminPaths,
 }: AdminNavbarProps) => {
   const isPaidPlan = useSelector(getIsPaidPlan);
+  const dispatch = useDispatch();
+
+  useRegisterShortcut(
+    [
+      {
+        id: "admin-change-tab",
+        perform: (_, event) => {
+          if (!event?.key) {
+            return;
+          }
+          const key = parseInt(event.key);
+          const path = adminPaths[key - 1]?.path;
+
+          if (path) {
+            dispatch(push(path));
+          }
+        },
+      },
+    ],
+    [adminPaths],
+  );
```

Comment:
🙁  this might not work so well with non data-based tabs, but we can cross that bridge if/when we get there